docs/api-reference/carto/overview.md

```diff
@@ -151,7 +152,12 @@ The CARTO submodule includes the CartoLayer that simplifies the interaction with
     });
     ```
 
-    The formats available are JSON, GEOJSON, TILEJSON, and NDJSON. [NDJSON](http://ndjson.org/) (Newline Delimited JSON) allows to handle incremental data loading https://deck.gl/docs/developer-guide/performance#handle-incremental-data-loading.
+    The formats available are JSON, GEOJSON, TILEJSON, and NDJSON. [NDJSON](http://ndjson.org/) (Newline Delimited JSON) allows to handle incremental data loading https://deck.gl/docs/developer-guide/performance#handle-incremental-data-loading. 
+
+* If the format is not explicitly specified, `getData` will pick a format automatically, depending on what is available from the CARTO API. The `getData` function returns a `Object` with the following properties:
```

Comment:
If possible, I'd add another level of indentation for clarity. The first indentation level should describe just the two options available (using API v3 or not)